src/composables/peerIdGenerator.js

```diff
@@ -0,0 +1,35 @@
+//A 10-character peer ID generator
+
+function generatePeerIdCharacter(number) {
```

Comment:
We don't need to input the number of characters it has to generate. we already know it's 10